safe_transaction_service/history/services/transaction_service.py

```diff
@@ -181,8 +182,8 @@ def get_all_tx_hashes(
         module_tx_ids = (
             ModuleTransaction.objects.filter(safe=safe_address)
             .annotate(
-                execution_date=F("internal_tx__ethereum_tx__block__timestamp"),
-                block=F("internal_tx__ethereum_tx__block_id"),
+                execution_date=F("internal_tx__timestamp"),
+                block=F("internal_tx__block_number"),
```

Comment:
I'm not sure but seems that the timestamp before was a block timestamp but now is internal tx timestamp. 
This is not going to affect the result?

Reply:
Because that table was refactored to also have timestamp, to optimize some queries, but I forgot to change this one 😉